### DISCUSSION TITLE
Welcome to Expenso Discussions!

### DISCUSSION THREAD

**Discussion by Spikeysanju** [Spikeysanju/Expenso#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dszvivian** [comment]
How can we give app apk  link in our readme file ??

**Reply by Spikeysanju** [reply]
@dszvivian Can you please elaborate?

**Reply by bhardwajprateek291200** [reply]
He wanted to say if we can give the latest release link of apk in the README for easy download.

**Reply by Spikeysanju** [reply]
It's already there in readme @dszvivian @bhardwajprateek291200 

![image](https://github.com/Spikeysanju/Expenso/assets/23400022/1504c96c-f378-4055-8cdd-4dc8edeb4bca)

**Comment by bhardwajprateek291200** [comment]
Hi @Spikeysanju,

Are we working on this actively?

Regards,
Prateek

**Comment by restora-veneration** [comment]
Hi, 
Is there a way to change the currency?
Good app regardless
